File: INICIANTE/1040 Média 3.py
n1, n2, n3, n4 = map(float, input().split())

media_total = ((n1 * 2) + (n2 * 3) + (n3 * 4) + (n4 * 1)) / 10

if media_total >= 7.0:
    print('Media: %.1f' % media_total)
    print('Aluno aprovado.')
elif media_total < 5.0:
    print('Media: %.1f' % media_total)
    print('Aluno reprovado.')
else:
    print("Media: %.1f" % media_total)
    print('Aluno em exame.')

    nova_nota = float(input())
    print('Nota do exame: %.1f' % nova_nota)
    media_final = (nova_nota + media_total) / 2

    if media_final >= 5.0:
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print('Media final: %.1f' % media_final)

File: C/INICIANTE/1040 Média 3.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    float n1, n2, n3, n4, media;
    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);
    media = ((n1*2 + n2*3 + n3*4 + n4*1) / 10);
    printf("Media: %.1f\n", media);
    if (media >= 7.0)
        printf("Aluno aprovado.\n");
    else if (media < 7.0 && media >= 5.0)
    {
        printf("Aluno em exame.\n");
        float n5;
        scanf("%f", &n5);
        printf("Nota do exame: %.1f\n", n5);
        if (((media + n5)/2) >= 5.0)
            printf("Aluno aprovado.\n");
        else
            printf("Aluno reprovado.\n");
        printf("Media final: %.1f\n", ((media + n5)/2));
    }
    else
        printf("Aluno reprovado.\n");
    return 0;
}
